webcompat/webhooks/model.py

```diff
@@ -234,6 +236,21 @@ def get_public_issue_number(self):
             url = self.public_url.strip().rsplit('/', 1)[1]
         return url
 
+    def classify(self):
+        """Make a request to bugbug and label the issue.
+
+        Gets issue classification from bugbug and labels
+        the issue if probability is high
+        """
+        data = get_issue_classification(self.number)
```

Comment:
This here might raise an Exception. 
https://github.com/webcompat/webcompat.com/pull/3573/files#diff-96859bd3238aaaffb2c63f7fd42e9899ba93c7e69bf96dd6c33f6de6f58b42feR266

but this is not handled here, with a try. 
Later on in https://github.com/webcompat/webcompat.com/pull/3573/files#diff-c2434db66ee7efbeb6ebcafb5470cc409f4a0cc61ac35709a32a1a0341c4b71dR306
the code is attempting to catch an HTTPError. That's good. 

but the exception before is very generic and not only HTTP
```
raise Exception(msg)
```

Reply:
I've changed it to be ConnectionError and added it to catch handle in https://github.com/webcompat/webcompat.com/pull/3573/commits/91cc36c48523bc7a1f328078fc2a283fc3253c48#diff-c2434db66ee7efbeb6ebcafb5470cc409f4a0cc61ac35709a32a1a0341c4b71dR303

Reply:
Wonderful.